frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/children/$childId/dental-insurance.tsx

```diff
@@ -191,7 +191,7 @@ export default function AccessToDentalInsuranceQuestion() {
               </Button>
               <ButtonLink
                 id="back-button"
-                routeId="$lang+/_public+/apply+/$id+/adult-child/review-child-information"
+                routeId="$lang+/_public+/apply+/$id+/adult-child/children/index"
```

Comment:
The cancel edit mode link to redirect to review child information. This change can be reverted.